apps/website/app/utils/getHtmlFromMarkdown.ts

```diff
@@ -0,0 +1,33 @@
+import { unified } from "unified";
+import remarkParse from "remark-parse";
+import remarkRehype from "remark-rehype";
+import rehypeStringify from "rehype-stringify";
+import { toString } from "mdast-util-to-string";
+import { visit } from "unist-util-visit";
+
+function remarkHeadingId() {
+  return (tree: any) => {
+    visit(tree, "heading", (node) => {
+      const text = toString(node);
+      const id = text
+        .toLowerCase()
+        .replace(/[^a-z0-9]+/g, "-")
+        .replace(/(^-|-$)/g, "");
+
+      node.data = {
+        hName: `h${node.depth}`,
+        hProperties: { id },
+      };
+    });
+  };
+}
+
+export async function getHtmlFromMarkdown(markdown: string): Promise<string> {
+  const htmlString = await unified()
+    .use(remarkParse)
+    .use(remarkHeadingId)
```

Comment:
majority of the change in this function is for this `remarkHeadingId`.

I need to pass in the `id` to section headers for a table of contents and was having trouble finding a good way to do it.  This refactor seems to satisfy for both `(blog)` and the upcoming `(docs)